fedimint-server/src/consensus/server.rs

```diff
@@ -442,7 +441,7 @@ impl ConsensusServer {
                 _ => break vec![],
             };
         };
-        let proposal = self.process_events_then_propose(override_proposal).await;
```

Comment:
On mobile, so can't test, but why doesn't this break the functionality that we can force a guardian to accept a specific epoch? See #2309

Reply:
I'm not familiar enough with the consensus code to have much input on that. I just noticed that the only place `override_proposal` was used, `None` was passed in. If there are plans to eventually use this for something, I can drop the commit!

Reply:
I think it's a leftover of forcing an epoch being moved to be admin api endpoint.

```
    // Save any API events we have in the channel then create a proposal
    async fn process_events_then_propose(
        &mut self,
        override_proposal: Option<ConsensusProposal>,
    ) -> ConsensusProposal {
        while let Some(Some(event)) = self.api_receiver.next().now_or_never() {
            match event {
                ApiEvent::ForceProcessOutcome(outcome) => self.force_process_epoch(outcome).await,
                event => {
                    self.consensus.api_event_cache.insert(event);
                }
            };
        }
        let consensus_proposal = self.consensus.get_consensus_proposal().await;
        self.consensus.api_event_cache.clear();
        override_proposal.unwrap_or(consensus_proposal)
    }
```

We now force an epoch as an event sent through `api_endpoint`, while the `override_proposal: Option<ConsensusProposal>,` never actually gets set to any `Some` on the server side. @elsirion